### ISSUE TITLE
Not working the checkbox of Search Unused Keywords results

### ISSUE THREAD

**Issue by HelioGuilherme66** [robotframework/RIDE#2300]
After doing a Search Unused Keywords, clicking on a result selects the keyword, but the checkbox selection is not functional, so the button to delete it, does not become active.

Observed on Linux and Windows.

Screenshot of experimental branch `colours` on my fork:
![image](https://user-images.githubusercontent.com/2866630/91908080-e3dd2700-eca2-11ea-9cb5-2d532467f719.png)

**Comment by HelioGuilherme66** [comment]
This problem does not happen with wxPython 4.0.7 on Fedora 32, Python 3.8.5.

**Comment by HelioGuilherme66** [comment]
Confirmed fixed on wxPython 4.1.1 on Fedora 32, Python 3.8.9.

```
20210514 00:59:09.203 [INFO]: Found Robot Framework version 4.0.2 from /usr/local/lib/python3.8/site-packages/robot.

20210514 00:59:09.204 [INFO]: Started RIDE v2.0b2.dev3 using python version 3.8.9 with wx version 4.1.1 in linux.

```
